src/extraction_output_manager/ingestibles.py

```diff
@@ -107,3 +110,40 @@ def add_datafiles(  # pylint: disable=missing-function-docstring
         datafiles: List[RawDatafile],
     ) -> None:
         self.datafiles.extend(datafiles)
+
+    def print(self, stream: TextIO, skip_datafiles: bool = True) -> None:
+        def write_header(text: str) -> None:
+            stream.write(f"\n\n{'=' * len(text)}\n")
+            stream.write(text)
+            stream.write(f"\n{'=' * len(text)}\n\n")
+
+        def write_dataclasses(models: Sequence[BaseModel]) -> None:
+            for model in models:
+                stream.write(model.model_dump_json(indent=4))
+                stream.write("\n")
+
+        write_header("Projects")
+        write_dataclasses(self.get_projects())
+
+        write_header("Experiments")
+        write_dataclasses(self.get_experiments())
+
+        write_header("Datasets")
+        write_dataclasses(self.get_datasets())
+
+        if not skip_datafiles:
+            write_header("Datafiles")
+            write_dataclasses(self.get_datafiles())
+
+    @staticmethod
```

Comment:
Nothing wrong with this approach but I wonder if we would do better to have an __init__ script that took optional lists of PEDD and created the class with the objects prefilled? Just a thought